src/stim/simulators/frame_simulator.test.cc

```diff
@@ -26,7 +26,7 @@ TEST(FrameSimulator, get_set_frame) {
     CircuitStats circuit_stats;
     circuit_stats.num_qubits = 6;
     circuit_stats.max_lookback = 999;
-    FrameSimulator sim(circuit_stats, FrameSimulatorMode::STORE_DETECTIONS_TO_MEMORY, 4, SHARED_TEST_RNG());
+    FrameSimulator<MAX_BITWORD_WIDTH> sim(circuit_stats, FrameSimulatorMode::STORE_DETECTIONS_TO_MEMORY, 4, SHARED_TEST_RNG());
```

Comment:
Use the macro `TEST_EACH_WORD_SIZE_W` instead of `TEST` (see `simd_word.test.cc` for examples), then replace `MAX_BITWORD_WIDTH` with `W`.